src/overlays/actors/ovl_En_Skb/z_en_skb.c

```diff
@@ -498,19 +499,19 @@ void EnSkb_Update(Actor* thisx, PlayState* play) {
     EnSkb* this = (EnSkb*)thisx;
     s32 pad;
 
-    func_80AFD968(this, play);
+    EnSkb_CheckDamage(this, play);
     Actor_MoveForward(&this->actor);
     Actor_UpdateBgCheckInfo(play, &this->actor, 15.0f, 30.0f, 60.0f,
                             UPDBGCHECKINFO_FLAG_0 | UPDBGCHECKINFO_FLAG_2 | UPDBGCHECKINFO_FLAG_3 |
                                 UPDBGCHECKINFO_FLAG_4);
     this->actionFunc(this, play);
     this->actor.focus.pos = this->actor.world.pos;
     this->actor.focus.pos.y += (3000.0f * this->actor.scale.y);
-    if (this->unk_281 != 0) {
+    if (this->hitboxActive != 0) {
```

Comment:
We avoid calling colliders "hitbox" because it's too vague for OoT's various colliders

`enableAT` would probably be okay

Also this member seems to be a boolean: you can replace the `= 0`, `= 1` with `= false`, `= true` and this condition can just be `if (this->enableAT)` (it usually doesn't break matching)